build_openroad.sh

```diff
@@ -20,10 +20,7 @@ OPENROAD_APP_BRANCH="master"
 INSTALL_PATH="$(pwd)/tools/install"
 
 YOSYS_USER_ARGS=""
-YOSYS_ARGS="\
-CONFIG=gcc \
```

Comment:
Removing the `CONFIG=gcc` seems to be causing some issue with clag on the CI.
`#8 0.412 /bin/sh: clang: command not found`
https://jenkins.openroad.tools/blue/organizations/jenkins/OpenROAD-flow-scripts-Public%2Fpublic_tests_all-pr/detail/PR-1468-head/6/pipeline/595/#step-600-log-2164